src/components/Popup.js

```diff
@@ -4,6 +4,7 @@ export default class Popup {
     this._handleEscClose = this._handleEscClose.bind(this);
     this._handleCloseOnClick = this._handleCloseOnClick.bind(this);
     this._closeButton = this._popupElt.querySelector(".modal__close-button");
+    this._saveButton = this._popupElt.querySelector(".modal__save-button");
```

Comment:
`this._saveButton` is not needed in `Popup` because the image modal doesn't have it. Universal classes can't have elements that don't exist in all modals